Sveltekit: App generation does not work OOTB

**Describe the bug**
After installing `@nxext/sveltekit@11.0.0-next.2, attempting to run `nx g @nxext/sveltekit:app <app_name>` will return an error:
`Cannot find module '@angular-devkit/schematics`. 

**To Reproduce**
Steps to reproduce the behavior:
1. Generate a new empty NX workspace (in my case, with TS): `npx create-nx-workspace@latest <workspace-name>` and choose `ts` from the list of templates (should create an empty workspace with JS/TS plugin installed)
1. Install latest SvelteKit NX Extension package: `yarn add -D @nxext/sveltekit`.
1. Attempt to generate new SvelteKit application: `nx g @nxext/sveltekit:app <app-name>`.

Error will be returned: `Cannot find module '@angular-devkit/schematics`. Note that installing `@angular-devkit/schematics` manually via yarn does appear to resolve the issue and then allows for generation of an application.

**Expected behavior**
Generating the application should work out of the box by default and not require manual installation of the `@angular-devkit/schematics` node package.

**Additional context**
FWIW, I tried this with two different new workspaces, one with `@nxext/svelte` already installed and one without and it didn't seem to make a difference.

--- a/packages/sveltekit/README.md
+++ b/packages/sveltekit/README.md
@@ -1,5 +1,37 @@
-# sveltekit
+# @nxext/sveltekit
 
 This library is experimental, be aware...
 
 Right now there are the `app` and `component` generators.
+
+## Getting Started
+
+Add this plugin to an existing NX workspace:
+
+```bash
+yarn add --dev @nxext/sveltekit
+```
+
+or
+
+```bash
+npm install @nxext/sveltekit --save-dev
+```
+
+Generate a new application:
+
+```bash
+nx g @nxext/sveltekit:app my-app
+```
+
+## Run your project
+
+> Note: you may see an error in your command line when you run any of the following commands with older versions of this extension: `config.kit.target is no longer required, and should be removed`. If you see this error, remove the `kit` object from your `svelte.config.js` file at the root of your app.
+
+- Run `nx serve my-app` to run the app locally (open http://localhost:3000 in your browser)
+- Run `nx test my-app` to test the app
+- Run `nx build my-app` to create a build of your app.
+
+## Documentation
+
+The full docs can be found [here](https://nxext.dev/docs/sveltekit/overview).

--- a/packages/sveltekit/src/generators/application/lib/install-dependencies.ts
+++ b/packages/sveltekit/src/generators/application/lib/install-dependencies.ts
@@ -1,5 +1,6 @@
 import { addDependenciesToPackageJson, Tree } from '@nrwl/devkit';
 import {
+  angularSchematicsVersion,
   svelteKitAdapterVersion,
   svelteKitVersion,
   sveltePreprocessVersion,
@@ -11,6 +12,8 @@ export function installDependencies(host: Tree) {
     host,
     {},
     {
+      // Resolves https://github.com/nxext/nx-extensions/issues/580
+      '@angular-devkit/schematics': angularSchematicsVersion,
       '@sveltejs/adapter-node': svelteKitAdapterVersion,
       '@sveltejs/kit': svelteKitVersion,
       svelte: svelteVersion,

--- a/packages/sveltekit/src/generators/utils/versions.ts
+++ b/packages/sveltekit/src/generators/utils/versions.ts
@@ -3,3 +3,4 @@ export const svelteVersion = '^3.44.0';
 export const svelteKitVersion = 'next';
 export const svelteKitAdapterVersion = 'next';
 export const sveltePreprocessVersion = '^4.9.4';
+export const angularSchematicsVersion = '^13.0.0';

